src/Ledger/Utxo.lagda

```diff
@@ -200,7 +214,7 @@ For all $\var{env}\in\UTxOEnv$, $\var{utxo},\var{utxo'}\in\UTxO$, $\var{fee},\va
 pov :
 \end{code}
 \begin{code}[inline*]
-  utxo ∩ᵖ outs tx ≡ ∅ᵐ
+  txid tx ∉ map proj₁ (dom (utxo ˢ))
```

Comment:
It would be great if it was clearer what the `proj₁` applies to. Sometimes it gets something from an existential, but sometimes it's used to get the first element of a pair. I had to look deeper here to understand what the `proj₁` applies to.

Reply:
Let's merge the PR then and turn this into an issue :)